### PR TITLE
Add GitHub CLI to course preparation

### PR DESCRIPTION
**Pull request by rulasg**
The actual pre requisites does not avoid issues with it authentication.

Managing this authentication failing during the course is complicated.

GitHub Cli has the ability to set and managed git authentication very easily. 

```sh
gh auth login
```

Adding the installation of GitHub CLI in the attendees computers allows an easy fixing for the teacher if needed with any attendee.

### CODE REVIEW

**Review by amyschoen — APPROVED** [review]
Thank you for your contribution.  This is a great addition.  😀

**Comment by rulasg** [comment]
@amyschoen Workflows waiting approval. Thanks.

**Comment by amyschoen** [comment]
Apologies.  I didn't realize the linter needed explicit approval.

File: docs/01_getting_ready_for_class.md [hunk 1]
```diff
@@ -101,6 +101,15 @@ Replace `YOUR_PROXY_USERNAME` with the username used to authenticate into your p
 
 For this class, we will use a basic text editor to interact with our code. Let's make sure you have one installed and ready to work from the command-line.
 
+### Step 5: Intstall GitHub Cli
+
+GitHub command-line tool has several funtionality that may be very useful during the labs setup. 
+Install GitHub CLI from [here](https://github.com/cli/cli#installation) and confirm that is available on your console.
```
Comment (parkerbxyz) [inline docs/01_getting_ready_for_class.md:107]:
```suggestion
### Step 5: Install GitHub CLI (optional)

The GitHub command-line tool has funtionality that may be useful during the labs setup. 

Install GitHub CLI from [here](https://github.com/cli/cli#installation), then confirm it is available on your console:
```

**Comment by rulasg** [comment]
@amyschoen @parkerbxyz Workflow waiting approval. THX

**Review by parkerbxyz — APPROVED** [review]
It just needs a new approving review since new commits were pushed.

**Pull request merged**